packages/core/src/runtime/utils/hashSequence.ts

```diff
@@ -17,6 +17,15 @@ function padEndHash(value: string): string {
   return value;
 }
 
-export function hashSequence(classes: string, dir: 'ltr' | 'rtl'): SequenceHash {
-  return SEQUENCE_PREFIX + padEndHash(hash(classes + dir));
+export function hashSequence(
+  classes: string,
+  dir: 'ltr' | 'rtl',
+  sequenceIds: (SequenceHash | undefined)[] = [],
+): SequenceHash {
+  return process.env.NODE_ENV !== 'production'
+    ? SEQUENCE_PREFIX +
+        padEndHash(hash(classes + dir)) +
+        DEBUG_SEQUENCE_SEPARATOR +
+        padEndHash(hash(sequenceIds.join('')))
+    : SEQUENCE_PREFIX + padEndHash(hash(classes + dir));
```

Comment:
nit, let's make this more readable

```suggestion
  if (process.env.NODE_ENV === 'production') {
    return SEQUENCE_PREFIX + padEndHash(hash(classes + dir));
  }

  return (
    SEQUENCE_PREFIX +
    padEndHash(hash(classes + dir)) +
    DEBUG_SEQUENCE_SEPARATOR +
    padEndHash(hash(sequenceIds.join('')))
  );
```